docs/index.rst

```diff
@@ -20,6 +20,21 @@ and the `Python Packaging User Guide`_ for user documentation. You can
 also join ``#pypa`` or ``#pypa-dev`` `on Freenode`_, or the `pypa-dev
 mailing list`_, to ask questions or get involved.
 
+Overview
+--------
+
+Twine is a command-line tool for interacting with PyPI securely over HTTPS. Its
+command line arguments are parsed in ``twine/twine/cli.py``. Currently, twine
```

Comment:
Would you be willing to share the reasoning around specifying the path as "twine/twine"? I think most people will be viewing those files as just "twine/cli.py" or "twine/commands/register.py"

Reply:
My reasoning for specifying the path as twine/twine is that is what the path is, including the `twine` directory that gets cloned from GitHub. If it's confusing to people, I can specify them as `twine/cli.py`.

Reply:
I see what you mean now. I think it's more common to do the following:

```
git clone https://github.com/pypa/twine
cd twine
# edit twine/cli.py
```

I think the custom that I've seen tends to refer to things from the root of the repository. But that certainly isn't the only way to work on projects and maybe it's better to be explicit that we are talking about working that way?